### PR TITLE
DOC-3704 updated start/stop and next steps

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-3704](https://redislabs.atlassian.net/browse/DOC-3704)
There was actually some inconsistency among all the platforms (they each had something useful that the others didn't) so I've tried to bring them into line.

[DOC-3704]: https://redislabs.atlassian.net/browse/DOC-3704?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dwdougherty** [comment]
I marked this PR as "do not merge yet" because I'd just started working on [a different ticket with the same ask](https://redislabs.atlassian.net/browse/DOC-3811). I want some time to complete my research, and, when I'm done, I'll add my findings to this PR.

**Comment by dwdougherty** [comment]
Here are my updates. The instructions for installing Redis Stack on Ubuntu (Jammy) using Snap will likely need to be changed, as the given instructions don't actually work.

Also, I think it would be worth your time to confirm steps on each listed OS using GCP VMs. I did the best I could to capture notes as I tested, but having a second pair of eyes on this would be great.

File: content/operate/oss_and_stack/install/install-stack/linux.md
Comment (dwdougherty) [inline content/operate/oss_and_stack/install/install-stack/linux.md:115]:
```suggestion
Follow these steps to integrate Redis Stack with `systemd` so you can start/stop in/from the background:
```

**Review by dwdougherty — APPROVED** [review]
@andy-stark-redis I'm approving without your final review so you're not held up.

File: content/operate/oss_and_stack/install/install-stack/linux.md
Comment (dwdougherty) [inline content/operate/oss_and_stack/install/install-stack/linux.md:153]:
```suggestion
{{< highlight bash  >}}
sudo systemctl start redis-stack-server
{{< / highlight  >}}

To stop the service, use:

{{< highlight bash  >}}
sudo systemctl stop redis-stack-server
{{< / highlight  >}}
```

**Comment by dmaier-redislabs** [comment]
@andy-stark-redis @dwdougherty 

We tested the Snap instructions from this docs change on Ubuntu today. There are some issues with the current instructions.

1. The snap package doesn't define a service, so you can't start/stop a service via snap in that case
2. The command in the docs uses the package name `redis` which is anyway wrong because the snap package that gets installed is `redis-stack-server`
3. Without a service and with having `redis-stack-server` installed, you should use sth. like `snap run redis-stack-server` to bring Redis Stack up

We also found an issue on Ubuntu machines that didn't have OpenSSL 1.1 installed. This might be related to the Ubuntu version. I think Vlad fixed this inside the package itself by adding libssl as a dependency.

I strongly recommend executing the actual installation commands by verifying if they work before merging installation documentation changes. You should also be careful copying and pasting instructions from the Redis installation instructions to the Redis Stack documentation. Please let us know if you find any issue with the packages themselves and we will take a look.

**Pull request merged**
